### PR TITLE
Extremely lazy fix for #138

### PR DESCRIPTION
**Pull request by sauyon**
This just uses $XDG_CONFIG_HOME/quodlibet as the default directory unless it doesn't exist and ~/.quodlibet does.

I'm willing to go through and actually put files where they should be if that's necessary.

EDIT: For #138.

### CODE REVIEW

File: quodlibet/quodlibet/_main.py [hunk 1]
```diff
@@ -118,7 +118,16 @@ def get_user_dir():
     if os.name == "nt":
         USERDIR = os.path.join(windows.get_appdate_dir(), "Quod Libet")
     else:
-        USERDIR = os.path.join(os.path.expanduser("~"), ".quodlibet")
+        if 'XDG_CONFIG_DIR' in environ:
```
Comment (lazka) [inline quodlibet/quodlibet/_main.py]: Use quodlibet.util.path.xdg_get_config_home

File: quodlibet/quodlibet/_main.py [hunk 2]
```diff
@@ -118,7 +118,16 @@ def get_user_dir():
     if os.name == "nt":
         USERDIR = os.path.join(windows.get_appdate_dir(), "Quod Libet")
     else:
```
Comment (lazka) [inline quodlibet/quodlibet/_main.py]: We don't want this change on macOS (quodlibet.util.is_osx), needs a new case using the old path for this.

**Comment by lazka** [comment]
Notice the code quality tests failing. You can run them locally using either "setup.py quality" or "py.test tests/quality"

**Comment by sauyon** [comment]
I'm seeing a bunch of quality errors from stuff in optpackages but I'm assuming I'm to ignore those; can I turn them off?

**Comment by lazka** [comment]
uh, optpackages should get skipped for this (in tests.quality.util.iter_project_py_files())

**Comment by sauyon** [comment]
It might be worth moving the check for QUODLIBET_USERDIR/build type above to speed up launches (slightly).

**Comment by sauyon** [comment]
Ah - it was finding them in `build/lib/quodlibet/optpackages`. Getting rid of that worked fine (maybe it should ignore `root/build` as well?).

**Comment by lazka** [comment]
Ah, right, thanks. I'll have a look.

**Comment by lazka** [comment]
Thanks!

**Pull request merged**